tests/test_client.py

```diff
@@ -819,15 +810,10 @@ def test_create_blank_record_with_baseid(client, user):
     assert res.status_code == 201
     rec = res.json
     assert rec["did"]
-    res = client.get("/index/" + rec["did"])
+    res = client.get("/index/?baseid=baseid_123")
```

Comment:
cool, please use `doc["base_id"]` instead of `baseid_123` here, so that the test doesn't break if someones change the data later